tests/test_sal_info.py

```diff
@@ -63,7 +63,7 @@ async def test_salinfo_constructor(self) -> None:
                 with pytest.raises(RuntimeError):
                     salinfo.assert_started()
 
-                asyncio.create_task(salinfo.start())
+                await asyncio.create_task(salinfo.start())
```

Comment:
Is this change right? I think the idea was to run it in the background (hence, no `await`) and than verify that the `start_task` is marked as done at the end.

Reply:
Thanks for pointing this out. I reverted the change.